modules/static/src/main/java/org/opencastproject/fsresources/StaticResourceServlet.java

```diff
@@ -92,7 +137,11 @@ public StaticResourceServlet() {
    */
   @Activate
   public void activate(ComponentContext cc) {
-    if (cc != null) {
+    if (cc == null) {
+      authRequired = true;
```

Comment:
Shouldn't this be `false` to preserve previous behaviour?  If not, then we need a note in the release docs, likely in big bold font.

Reply:
Yep, this needs to be in the release docs. This may even deserve a section in the migration notes. Nevertheless, this should be `true` since this is likely what people expect: If I use the admin interface to define that only `ROLE_XY` should have access, that's what I expect file access is checked for.

Except for a few special cases, most adopters I know indirectly expect this already so changing this shouldn't be a problem for them. New adopters certainly expect that. For the few *special cases*, it's easy enough to change.

Reply:
Just went through a few systems in my head and I couldn't remember anyone for which it should be a problem. At least, I'm reasonably sure that it wouldn't be a problem for UOS, SWITCH, ETH, Graz, Wismar, Bochum, Vechta, Oldenburg, Halle, Hannover, Ulm, HS Osna, … I'm stopping here ;-)